src/main.js

```diff
@@ -35,6 +35,23 @@ class ameClient {
             throw err;
         }
     }
+    
+
+    async image(endpoint){
```

Comment:
Can you add parameters because in the near future the API will use more options

Currently the API at wallpaper level with an nsfw parameter

```suggestion
    async image(endpoint, data ={}){
```